bis-rules/package.json

```diff
@@ -28,47 +28,36 @@
     "url": "http://www.bentley.com"
   },
   "devDependencies": {
-    "@itwin/build-tools": "^3.7.5",
-    "@itwin/eslint-plugin": "^3.7.5",
-    "@itwin/ecschema-metadata": "^3.7.5",
-    "@types/chai": "^4.1.7",
+    "@itwin/build-tools": "^4.1.0",
+    "@itwin/eslint-plugin": "4.0.0-dev.44",
+    "@itwin/ecschema-metadata": "^4.1.0",
+    "@types/chai": "4.3.1",
     "@types/mocha": "^8.2.2",
     "@types/rimraf": "^2.0.2",
     "@types/sinon": "^7.0.5",
     "ts-node": "^7.0.1",
-    "chai": "^4.2.0",
+    "chai": "^4.1.2",
```

Comment:
@naveedkhan8067 Assuming reverting the chai version was for a reason?

Reply:
I was following what itwinjs-core packages are using that's why I reverted it :)

Reply:
OK